### PR TITLE
Add tests for standard cluster roles and bindings

### PR DESCRIPTION
**Pull request by markxnelson**

### CODE REVIEW

File: tests/e2e/pkg/common.go [hunk 1]
```diff
@@ -220,6 +220,17 @@ func Jq(node interface{}, path ...string) interface{} {
 	return node
 }
 
+// SliceContainsString checks if the input slice (an array of strings)
+// contains an entry which matches the string s
+func SliceContainsString(slice []string, s string) bool {
```
Comment (pfmackin) [inline tests/e2e/pkg/common.go:225]: Cnce we have top level go.mod this should be put into top level pkg/vzstring/...  or equiv. There are a few of these types of functions scattered throughout

File: tests/e2e/pkg/kubernetes.go [hunk 2]
```diff
@@ -340,6 +341,58 @@ func DeleteNamespace(name string) error {
 	return err
 }
 
+// DoesClusterRoleExist returns whether a cluster role with the given name exists in the cluster
+func DoesClusterRoleExist(name string) bool {
```
Comment (pfmackin) [inline tests/e2e/pkg/kubernetes.go:345]: same comment, we have some of these functions already in platform-operator/test/...

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go [hunk 3]
```diff
@@ -20,4 +20,259 @@ var _ = ginkgo.Describe("Verrazzano", func() {
...
+			}
+
+			gomega.Expect(foundReadRule).To(gomega.BeTrue(),
+				"should be a rule that allows get,list,watch verbs")
+			gomega.Expect(foundWriteRule).To(gomega.BeTrue(),
+				"should be a rule that allows put,post verbs")
+		})
+	})
+
+	ginkgo.Describe("ClusterRole verrazzano-project-admin", func() {
```
Comment (pfmackin) [inline tests/e2e/verify-install/verrazzano/verrazzano_test.go:141]: looks like you check ClusterRole verrazzano-project-admin in 2 Describe blocks.

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go [hunk 4]
```diff
@@ -20,4 +20,259 @@ var _ = ginkgo.Describe("Verrazzano", func() {
...
+				}
+			}
+
+			gomega.Expect(foundReadRule).To(gomega.BeTrue(),
+				"should be a rule that allows get,list,watch verbs")
+			gomega.Expect(foundWriteRule).To(gomega.BeTrue(),
+				"should be a rule that allows put,post verbs")
+		})
+	})
+
```
Comment (pfmackin) [inline tests/e2e/verify-install/verrazzano/verrazzano_test.go:90]: I think ClusterRole verrazzano-monitor is missing

File: tests/e2e/pkg/common.go (hunk 1 above)
Comment (markxnelson) [reply tests/e2e/pkg/common.go:225]: we do not want tests to depend on product code

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go (hunk 4 above)
Comment (markxnelson) [reply tests/e2e/verify-install/verrazzano/verrazzano_test.go:90]: thanks, fixed that!

**Pull request closed without merging**